scripts/monitoring/run_auto_start_or_stop_workers.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+path=$PWD
+auth_token=''
+api_host_url='https://staging.eval.ai'
```

Comment:
We shouldn't be hardcoding this?

Reply:
This is just the default argument. When the script is deployed as crontab we will add separate endpoints as shell script params in crontab

Reply:
let's keep it blank then with the comments to avoid any future confusion.

Reply:
@Ram81 Can you please make this change.